RecoMuon/MuonIdentification/src/MuonCosmicCompatibilityFiller.cc

```diff
@@ -294,42 +294,16 @@ bool MuonCosmicCompatibilityFiller::isOverlappingMuon(const edm::Event& iEvent,
       reco::TrackRef outertrack = muon.outerTrack();
       reco::TrackRef costrack = cosmicMuon->outerTrack();
 
-      bool isUp = false;
-      if (outertrack->phi() > 0)
-        isUp = true;
-
       // shared hits
       int RecHitsMuon = outertrack->numberOfValidHits();
-      int RecHitsCosmicMuon = 0;
       int shared = 0;
       // count hits for same hemisphere
```

Comment:
This comment is not true any more now, and it could be also removed (otherwise it can be confusing for who's reading the code). Probably also a few of the remained commented out parts of the code, and for sure the definitions of  `muonRefVtx` and `cosmicRefVtx` in https://github.com/cms-sw/cmssw/blob/72272eae0f991f33d116a2f0071526ba7df1d1aa/RecoMuon/MuonIdentification/src/MuonCosmicCompatibilityFiller.cc#L315-L316

On the other hand, I wonder whether the original intention of the authors was to let some commented out code available for some special user purpose: in that case the  removed parts should be left in the code commented out, instead. Could someone from @cms-sw/muon-pog-l2 have a look and comment?